tests/012-test-container-root-lv-options.sh

```diff
@@ -0,0 +1,74 @@
+source $SRCDIR/libtest.sh
+
+# Test CONTAINER_ROOT_LV_NAME, CONTAINER_ROOT_LV_MOUNT_PATH and CONTAINER_ROOT_LV_SIZE directives.
+# Returns 0 on success and 1 on failure.
+test_container_root_volume() {
+  local devs=$TEST_DEVS
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local container_root_lv_name="container-root-lv"
+  local container_root_lv_mount_path="/var/lib/containers"
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  # Create config file
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+DEVS="$devs"
+VG=$vg_name
+CONTAINER_ROOT_LV_NAME=$container_root_lv_name
+CONTAINER_ROOT_LV_MOUNT_PATH=$container_root_lv_mount_path
+EOF
+
+ # Run docker-storage-setup
+ $DSSBIN >> $LOGS 2>&1
+
+ # Test failed.
+ if [ $? -ne 0 ]; then
+    echo "ERROR: $testname: $DSSBIN Failed." >> $LOGS
+    cleanup_all $vg_name $container_root_lv_name $container_root_lv_mount_path $devs
+    return $test_status
+ fi
+
+  # Make sure $CONTAINER_ROOT_LV_NAME {container-root-lv} got created
+  # successfully.
+  if ! lv_exists "$vg_name" "$container_root_lv_name"; then
+    echo "ERROR: $testname: Logical Volume $container_root_lv_name does not exist." >> $LOGS
+    cleanup_all $vg_name $container_root_lv_name $container_root_lv_mount_path $devs
+    return $test_status
+  fi
+
+  # Make sure $CONTAINER_ROOT_LV_NAME {container-root-lv} is
+  # mounted on $CONTAINER_ROOT_LV_MOUNT_PATH {/var/lib/containers}
+  local mnt
+  mnt=$(findmnt -n -o TARGET --first-only --source /dev/${vg_name}/${container_root_lv_name})
+  if [ "$mnt" != "$container_root_lv_mount_path" ];then
+   echo "ERROR: $testname: Logical Volume $container_root_lv_name is not mounted on $container_root_lv_mount_path." >> $LOGS
+   cleanup_all $vg_name $container_root_lv_name $container_root_lv_mount_path $devs
+   return $test_status
+  fi
+
+  cleanup_all $vg_name $container_root_lv_name $container_root_lv_mount_path $devs
```

Comment:
We will have to pass `"$devs" I think. Otherwise this will be a problem if more than one devices are mentioned in DEVS. 

DEVS="/dev/vdb /dev/vdc"

Checks rest of the tests are using quotes around devs when passing around.